docs/dev_guide/maintainer_workflow.rst

```diff
@@ -0,0 +1,240 @@
+.. _maintainer-workflow:
+
+Workflow for Maintainers
+========================
+
+This page is for maintainers |emdash| those of us who merge our own or other
+peoples' changes into the upstream repository.
+
+Being as how you're a maintainer, you are completely on top of the basic stuff
+in :ref:`development-workflow`.
+
+Integrating changes via the web interface (recommended)
+-------------------------------------------------------
+
+Whenever possible, merge pull requests automatically via the pull request manager on GitHub. Merging should only be done manually if there is a really good reason to do this!
+
+Make sure that pull requests do not contain a messy history with merges, etc. If this is the case, then follow the manual instructions, and make sure the fork is rebased to tidy the history before committing.
```

Comment:
Should this not be "rebase merge on Github"?

Reply:
check out on a test repo what it does, as I recall it rebases and stashes everything on top of master in a single commit without producing a merge commit. E.g. it's not what we prefer in astropy as it makes managing the backportings a real pain.